tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/config/TimeoutConfigBean.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package org.eclipse.microprofile.fault.tolerance.tck.config;
+
+import org.eclipse.microprofile.faulttolerance.Timeout;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.time.temporal.ChronoUnit;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Suite of methods for testing the various parameters of Timeout
+ */
+@ApplicationScoped
+public class TimeoutConfigBean {
+    @Timeout(value = 1, unit = ChronoUnit.MILLIS)
```

Comment:
Since `@Timeout` has a lot of advantages when used together with `@Asynchronous`, I am wondering add `@Asynchronous` on one of the endpoint might be good. Thoughts?

Reply:
In the config test?

Reply:
We're only testing timeout config here. Do you think we need a specific test for config when `@Timeout` and `@Asynchronous` are used together?

In general I prefer tests to remain focused. This class just tests Timeout + config while we have other tests for Timeout + async.

Reply:
Do we have Timeout + async + config? The reason is that timeout is better off to be used with `@Asynchronous`. We need to ensure the config works. I will be ok to skip `Timeout` with config and vote for `Timeout`+`Asynchronous` with config.

Reply:
> timeout is better off to be used with @Asynchronous

Why is that?

> We need to ensure the config works

We do. This PR does that. `@Asynchronous` makes no difference whatsoever.

Reply:
(Actually it might make a difference, depending on implementation, but so can putting the annotation on a class, or configuring an annotation parameter for all methods in a class, or configuring an annotation parameter globally. We decided to stick to this form of testing and I don't quite see how `@Asynchronous` is so very relevant.)

Reply:
In any case, I have just changed one test from synchronous to `@Asynchronous`.